content/source/docs/enterprise-beta/api/policies.html.md

```diff
@@ -0,0 +1,256 @@
+---
+layout: enterprise2
+page_title: "Policies - API Docs - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-api-policy"
+---
+
+# Policies API
+
+-> **Note**: These API endpoints are in beta and may be subject to change.
+
+[Sentinel Policy as Code](/docs/enterprise-beta/sentinel/index.html) is an embedded policy as code framework integrated with Terraform Enterprise. Policies are defined in organizations and enforced on all workspace Runs between a plan and an apply. A plan must pass the policy in order to proceed to the apply step. This doc covers operations to create, read, update, and delete the Sentinel Policies in an organization. The [Runs API](/docs/enterprise-beta/api/run.html) covers the steps for reading and overriding policy checks in a workspace run.
+
+
+## Create a Policy
+
+This endpoint enables you to create a policy and associate it with an organization.
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /organizations/:organization/policies |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - specifies the organization name where the Policy will be defined
+- `name` (`string: <required>`) - specifies the name of the policy. Accepts alphanumeric characters, as well as `-` and `_`. Cannot be modified after creation.
+- `mode` (`string: "soft"`) - specifies the enforcement level of the policy, it can be `hard`, `soft`, or `advisory`. Enforcement level details can be found in the [Managing Policies](/docs/enterprise-beta/sentinel/manage-policies.html) documentation.
```

Comment:
The `name` and `mode` are not top-level fields, and it's a bit confusing to have them laid out like this. It looks like this is suggesting that `name` and `mode` can both just be passed directly but that isn't the case - they are nested within the attributes differently like you have below. Not sure how to make that more clear, but I am wondering if Parameters is not the right place to have these. Perhaps just a more paragraph-style explanation of the payload below or something would be less misleading.

Reply:
Also if this is indicating the default is `soft`, that's wrong - it's actually `hard-mandatory` by default.